test/integration/offline.js

```diff
@@ -13,6 +13,7 @@ PouchDB.plugin(require('pouchdb-adapter-memory'))
 // environment, so we declare it in global scope here.
 global.PouchDB = PouchDB
 global.pouchdbFind = pouchdbFind
+global.navigator = undefined
```

Comment:
Why do you need to instantiate `gobal.navigator` to `undefined`.
If you don't declare `global.navigator`, then `global.navigator` is `undefined`...